samples/04_gis_analysts_data_scientists/temperature_forecast_using_time_series_data.ipynb

```diff
@@ -69,7 +69,7 @@
   },
```

Comment:
<p>Please move the <code>eng_interpolated_temp.zoom = 6</code>line to a cell below the rendered map.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/2162/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='85'/>